### PR TITLE
Update static page

### PR DESCRIPTION
**Pull request by QuangVinh93**

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Home" %>
+<div class="container-fluid text-center">
+  <div class="row content">
+    <div class="col-sm-2"></div>
+    <div class="col-sm-10 text-left"> 
+      <h1>Welcome to Framgia E-learing system</h1>
```
Comment (vietmt) [inline app/views/static_pages/home.html.erb]: i18n

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -12,4 +12,6 @@
  *
  *= require_tree .
  *= require_self
+ *= require bootstrap
  */
+
```
Comment (vietmt) [inline app/assets/stylesheets/application.css]: end file

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>BaseApp</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <title><%= t "view.layouts.title" %></title>
+  <%= stylesheet_link_tag "application", media: "all", 
+    "data-turbolinks-track" => true %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: "data-turbolinks-track":true %>

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: <%= javascript_include_tag "application", "data-turbolinks-track": true %>

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      </button>
+      <a class="navbar-brand" href="#"><%= t "views.layouts.logo" %></a>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="#"><%= t "views.layouts.home" %></a></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+          <a href="#">
```
Comment (hellovietnam93) [inline app/views/layouts/_header.html.erb]: sử dụng link_to

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>BaseApp</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <title><%= t "views.layouts.title" %></title>
+  <%= stylesheet_link_tag "application", media: "all",
+    "data-turbolinks-track" => true %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: "data-turbolinks-track": true %>

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,14 +1,23 @@
...
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: <%= javascript_include_tag "application", "data-turbolinks-track" : true %>

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,14 +1,23 @@
...
 <body>
-
-<%= yield %>
-
+  <%= render "layouts/header" %>
+  <div class="container">
+    <% flash.each do |message_type, message| %>
+      <div class="alert alert-<%= message_type %>"><%= message %></div>
+    <% end %>
+  <%= yield %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: ident

File: app/views/static_pages/home.html.erb [hunk 9]
```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Home" %>
```
Comment (hellovietnam93) [inline app/views/static_pages/home.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+      </button>
+      <%= link_to t("views.layouts.logo"), "#", class: "navbar-brand" %>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><%= link_to t("views.layouts.home"), "#" %></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+          <%= link_to "#" do %>
```
Comment (hellovietnam93) [inline app/views/layouts/_header.html.erb]: <%= link_to t("views.layouts.login"), "#", class="glyphicon glyphicon-log-in" do %>

File: app/assets/stylesheets/custom.css.scss [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+  background-color: #f1f1f1;
+  height: 100%;
+}
+
+@media screen and (max-width: 767px) {
+  .sidenav {
+    height: auto;
+    padding: 15px;
+  }
+  .row.content {height:auto;}
```
Comment (hellovietnam93) [inline app/assets/stylesheets/custom.css.scss]: xuống dòng đi

**Pull request merged**